ctest-next/src/template.rs

```diff
@@ -291,4 +458,139 @@ impl<'a> TranslateHelper<'a> {
 
         Ok(self.generator.map(item))
     }
+
+    /// Recursively get the properly mapped type for some syn::Type.
```

Comment:
Slight disambiguation

```suggestion
    /// Get the properly mapped type for some `syn::Type`, recursing for pointer types as needed.
```